v-next.sql

```diff
@@ -1,3 +1,8 @@
+\i ocd-3519.sql
+\i ocd-3554.sql
+\i ocd-3410.sql
+\i ocd-3574.sql
```

Comment:
This should only show one addition to this file and it should be your ocd-3574.sql file. Only tickets in STG should have ocd-* files here and any ticket that's been deployed to prod should not be here.